### PR TITLE
[dev-v2.7] rancher-monitoring 102.0.4+up40.1.2 update

### PR DESCRIPTION
**Pull request by alexandreLamarre**
un-rc rancher-monitoring to version `102.0.4+up40.1.2`

### CODE REVIEW

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

**Review by mallardduck — APPROVED** [review]
LGTM

File: release.yaml [hunk 1]
```diff
@@ -19,8 +19,12 @@ rancher-logging:
 rancher-logging-crd:
   - 102.0.3+up3.17.10
 rancher-monitoring:
-  - 102.0.4-rc1+up40.1.2
+  - 102.0.4+up40.1.2
 rancher-monitoring-crd:
-  - 102.0.4-rc1+up40.1.2
+  - 102.0.4+up40.1.2
+rancher-cis-benchmark:
```
Comment (mallardduck) [inline release.yaml:25]: Hmm. Actually maybe something off since validation failed.

Comment (mallardduck) [reply release.yaml:25]: Actually realized this is just friday brain since the "thumbs up validation" is failing not the one that checks yaml.

**Pull request merged**